iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/MergedLine.java

```diff
@@ -169,6 +171,11 @@ public String getId() {
         return id;
     }
 
+    public MergedLine setId(String id) {
+        this.id = id;
+        return this;
+    }
```

Comment:
Id is not mutable

Reply:
In LineAdderAdapter, in case of Line creation from both network.
A MergedLine is created with generated id from both DangingLine.
But i need to overrride generated id with the requested id by LineAdderAdapter::setId.

Reply:
I still disagree: I have to look deeper in the code to understand why the MergedLine can be created without having the final ID

Reply:
setId method must be private-package